docs/89_glossary.md

```diff
@@ -126,9 +126,69 @@ following requirements:
   * Something you know (Example: passwords)
   * Something you are (Example: biometrics)
 
-* **wrong outcome** – When the reported outcome does not match the
-actual outcome. This can happen due to various reasons like
-adjudication or equipment errors.
+* **wrong outcome** - When the reported outcome does not match the
```

Comment:
It might be worth clarifying if "wrong outcome" refers to only a _contest_ outcome, or if we will freely use it to refer to an "election" or some other collection of contests whose outcomes are simultaneously under consideration.